src/components/modules/toolbar/inline.ts

```diff
@@ -480,7 +480,7 @@ export default class InlineToolbar extends Module<InlineToolbarNodes> {
     /**
      * Get icon or title for dropdown
      */
-    const toolboxSettings = currentBlock.activeToolboxEntry || {};
+    const toolboxSettings = await currentBlock.getActiveToolboxEntry() || {};
```

Comment:
there's one problem with setting toggler's content this way: if none of the configured toolbox entries is active, toggler will contain only tool name

https://user-images.githubusercontent.com/31101125/168940401-759ee229-ac7c-4513-bbc0-5d37012bee1c.mov

Reply:
I think, it is ok because other titles and icons does not correctly fit here.